runtime/src/cost_model.rs

```diff
@@ -538,7 +514,8 @@ mod tests {
         let key1 = Pubkey::new_unique();
         let cost1 = 100;
         let cost2 = 200;
-        let updated_cost = (cost1 + cost2) / 2;
+        // updated_cost = (mean + 2*std) = 150 + 2 * 50 = 250
+        let updated_cost = 250;
```

Comment:
(mean + 2*std) could exceed Max of data points (in this case [100, 200]), is it necessary to cap it at max like `min( Max, (mean + 2*std) )`. IMO it isn't necessary, if we really want, can adjust down to `1.5 * std`. wdyt?